runtime/runtime.c

```diff
@@ -226,84 +224,5 @@ void* allocate_n_bytes_ptr(u32 n) {
     return get_memory_ptr_for_runtime(addr, n);
 }
 
-// FIXME: Currently the timer support stuff is disabled, pending cortex_m results
-// this provides a way to add a timeout for wasm execution, and support for that
-// TODO: Add a way for awsm to give us this value
-//WEAK unsigned int wasm_execution_timeout_ms = 0;
-//sigjmp_buf timeout_jump;
-//#define JUMPING_BACK 0xBACC
-
-// Precondition: you've already loaded timeout_jump with where we should jump after the timeout
-//void handle_sigalarm(int sig) {
-//    // TODO: We use siglongjmp which resets signal stuff, so perhaps this call is unnessesary
-//    signal(sig, SIG_IGN);
-//    siglongjmp(timeout_jump, JUMPING_BACK);
-//}
-//
-//void schedule_timeout() {
-//    signal(SIGALRM, handle_sigalarm);
-//    ualarm(wasm_execution_timeout_ms * 1000, 0);
-//}
-//
-//void cancel_timeout() {
-//    ualarm(0, 0);
-//}
-
 // If we are using runtime globals, we need to populate them
 WEAK void populate_globals() {}
```

Comment:
OT: Runtime globals turned out to be a garbage feature, I legitimately can think of no use for them. We should make an issue to review removing them

Reply:
Opened as #40 !